### PR TITLE
update rule sles-15-040250

### PR DESCRIPTION
**Pull request by svet-se**
#### Description:

- _Update rule file_permissions_sshd_private_key_

#### Rationale:

- _Accoring DISA recommendations Version 1, Release 9 from 26 January 2023 about SLE 15 STIG - "SLES-15-040250 - Updated text to reflect revised SSH key permissions guidance from vendor."_

### CODE REVIEW

**Comment by svet-se** [comment]
hey @marcusburghardt 

Thanks for the advice, I've rebased to master.

Have a nice day!

File: linux_os/guide/services/ssh/file_permissions_sshd_private_key/ansible/shared.yml [hunk 1]
```diff
@@ -7,8 +7,16 @@
...
+{{% set find_command = find_command_base + ' -group root -perm /u+xs,g+xws,o+xwrt' %}}
+{{% set permissions_mode = 'u-xs,g-xws,o-xwrt' %}}
+{{% else %}}
+{{% set find_command = find_command_base + ' -group root -perm /u+xs,g+xwrs,o+xwrt' %}}
+{{% set permissions_mode = 'u-xs,g-xwrs,o-xwrt' %}}
+{{% endif %}}
+
 - name: Find root:root-owned keys
-  command: '{{{ find_command_base }}} -group root -perm /u+xs,g+xwrs,o+xwrt'
+  command: '{{{ find_command }}}'
```
Comment (marcusburghardt) [inline linux_os/guide/services/ssh/file_permissions_sshd_private_key/ansible/shared.yml:19]: Please, use the FQCN for Ansible modules in all tasks: https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible

I know it was not part of the PR, but it would be great to take this opportunity to update the Ansible Playbook.

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

File: linux_os/guide/services/ssh/file_permissions_sshd_private_key/ansible/shared.yml (hunk 1 above)
Comment (svet-se) [reply linux_os/guide/services/ssh/file_permissions_sshd_private_key/ansible/shared.yml]: Thanks, @marcusburghardt ,

I've made the suggested updates and also a small refactoring of my changes.

Best regards

**Review by marcusburghardt — APPROVED** [review]

**Comment by ggbecker** [comment]
I see Automatus tests were not applicable:

```
Run tests/test_rule_in_container.sh --no-remove-machine-only --dontclean --logdir logs_bash --remediate-using bash --name ssg_test_suite --datastream $DATASTREAM file_permissions_sshd_private_key
Setting console output to log level INFO
INFO - The base image option has been specified, choosing Podman-based test environment.
INFO - Logging into logs_bash/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_file_permissions_sshd_private_key
WARNING - Script unfixable.fail.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_file_permissions_sshd_private_key evaluation resulted in notapplicable
WARNING - Script altcorrect_permissions.pass.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_file_permissions_sshd_private_key evaluation resulted in notapplicable
WARNING - Script multiple_keys.fail.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_file_permissions_sshd_private_key evaluation resulted in notapplicable
WARNING - Script lenient_permissions.fail.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_file_permissions_sshd_private_key evaluation resulted in notapplicable
WARNING - Script supercompliance.pass.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_file_permissions_sshd_private_key evaluation resulted in notapplicable
WARNING - Script altlenient_permissions.fail.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_file_permissions_sshd_private_key evaluation resulted in notapplicable
WARNING - Script correct_permissions.pass.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_file_permissions_sshd_private_key evaluation resulted in notapplicable
```

This PR is what most likely is causing: 

https://github.com/ComplianceAsCode/content/issues/10594
https://github.com/ComplianceAsCode/content/issues/10592
